tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -525,6 +525,39 @@ Options=mode=1777,strictatime,nosuid,nodev
 EOF
 	fi
 
+	if ! [ -f "./etc/systemd/system/run.mount" ] && \
+		! [ -f "./usr/lib/systemd/system/run.mount" ] &&
+		[ "$AGENT_INIT" != "yes" ]; then
+		local unitFile="./etc/systemd/system/run.mount"
+		info "Install run.mount in ./etc/systemd/system"
+		mkdir -p `dirname "$unitFile"`
+		cp ./usr/share/systemd/run.mount "$unitFile" || cat > "$unitFile" << EOF
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+#[Unit]
+#Description=Temporary Directory (/tmp)
+#Documentation=man:hier(7)
+#Documentation=https://www.freedesktop.org/wiki/Software/systemd/APIFileSystems
+#ConditionPathIsSymbolicLink=!/tmp
+#DefaultDependencies=no
+#Conflicts=umount.target
+#Before=local-fs.target umount.target
+#After=swap.target
```

Comment:
Good catch, I got the wrong commit, let me fix to the mount file iv used.

Reply:
Hi. I thought this would be a quick fix, but am still trying to get the mount file working again. Should I take the PR down in the interim? My bad about this.

Reply:
Nops, let me just mark this one as a "wip".  Let me know whenever it's "ready". :-)

Reply:
Alright, now ready. Sorry for my messy process on this one. 
My testing looks like starting a pod with an image built with this, and looking at `/proc/mounts` to compare the options for /run with an initrd boot.
To accomplish this there will be changes in the filesystem to the `/etc/fstab` and `/etc/systemd/system/basic.target.wants/kata-containers.target`. The goal of the kata target change is to make sure that the right systemd process starts, which can be checked with `systemctl status systemd-remount-fs.service`